en/02_Developer_Guides/08_Performance/06_ORM.md

```diff
@@ -48,6 +48,9 @@ SilverStripe\ORM\Connect\DBSchemaManager:
   check_and_repair_on_build: false
```

Comment:
I haven't renamed this to 05_ORM.md as github wanted to delete the old file and create a new file, rather than simply saying it's renamed. This would have made peer review very difficult, so I simply left a "gap" in the page numbering